src/TypeScript/EslintBridgeClient/EslintBridgeHttpWrapper.cs

```diff
@@ -34,40 +33,34 @@ namespace SonarLint.VisualStudio.TypeScript.EslintBridgeClient
 {
     internal interface IEslintBridgeHttpWrapper : IDisposable
     {
-        Task<string> PostAsync(string serverEndpoint, object request, CancellationToken cancellationToken);
+        Task<string> PostAsync(Uri serverEndpoint, object request, CancellationToken cancellationToken);
     }
 
-    [Export(typeof(IEslintBridgeHttpWrapper))]
-    [PartCreationPolicy(CreationPolicy.Shared)]
     internal sealed class EslintBridgeHttpWrapper : IEslintBridgeHttpWrapper
     {
-        private readonly IEslintBridgeProcess eslintBridgeProcess;
         private readonly ILogger logger;
         private readonly HttpClient httpClient;
 
-        [ImportingConstructor]
-        public EslintBridgeHttpWrapper(IEslintBridgeProcess eslintBridgeProcess, ILogger logger)
-            : this(eslintBridgeProcess, new HttpClientHandler(), logger)
+        public EslintBridgeHttpWrapper(ILogger logger)
+            : this(new HttpClientHandler(), logger)
         {
         }
 
-        internal EslintBridgeHttpWrapper(IEslintBridgeProcess eslintBridgeProcess, HttpMessageHandler httpHandler, ILogger logger)
+        internal EslintBridgeHttpWrapper(HttpMessageHandler httpHandler, ILogger logger)
         {
-            this.eslintBridgeProcess = eslintBridgeProcess;
             this.logger = logger;
             httpClient = new HttpClient(httpHandler);
         }
 
-        public async Task<string> PostAsync(string serverEndpoint, object request, CancellationToken cancellationToken)
+        public async Task<string> PostAsync(Uri serverEndpoint, object request, CancellationToken cancellationToken)
         {
             try
             {
                 var serializedRequest = request == null ? string.Empty : JsonConvert.SerializeObject(request, Formatting.Indented);
                 logger.LogDebug(Resources.INFO_RequestDetails, serverEndpoint, serializedRequest);
 
-                var port = await eslintBridgeProcess.Start();
```

Comment:
As we discussed offline, the wrapper is now dumb and accepts the given uri, without starting the process.
JavaScriptAnalyzer is the one that will call `eslintBridgeProcess.Start();`